_guides/project-boards/configure.md

```diff
@@ -285,11 +287,81 @@ The project board will automatically refresh.  The cards in the completed column
 <p>You may have noticed that we didn't provide an alternate template for cards that aren't completed.  If our custom template is blank, Cerb uses its own default template.</p>
 </div>
 
+### Using conditional logic with tasks
+
+Not every task is completed on time; what if we want to add a visual cue when a task is  past due?
+
+1. On that same profile go ahead and click on the gear button (⚙️ Edit) at the top of the profile page.
```

Comment:
Remove "go ahead". It could just read `"Click the **Edit** button again on the same project board profile from above."`